signalling/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3
+
+RUN pip3 install --user websockets
+
+WORKDIR /opt/
+COPY . /opt/
+
+RUN sed -i 's/sslctx.load_cert_chain(chain_pem, keyfile=key_pem)/pass/g' \
+    ./simple-server.py
+RUN sed -i 's/ssl=sslctx,//g' \
+    ./simple-server.py
```

Comment:
Maybe you want to add an option to `simple-server.py` to not use ssl instead? If you set `ssl=None` when calling `serve()`, it's the same as not passing an sslctx.